internal/services/topic.go

```diff
@@ -34,6 +35,13 @@ type TopicService struct {
 	initialized  bool
 }
 
+// ErrExpectedClusterSize defines the error raised when the expected cluster size is not met
+type ErrExpectedClusterSize struct{}
+
+func (e *ErrExpectedClusterSize) Error() string {
+	return fmt.Sprintf("Current cluster size doesn't met the expected one")
```

Comment:
```suggestion
	return fmt.Sprintf("Current cluster size differs from the expected size")
```

Should we include the values too?